### ISSUE TITLE
Safari version

### ISSUE THREAD

**Issue by Muffo** [boukestam/inbox-in-gmail#33]
Any plan to release this extension for Safari?

**Comment by russelldc** [comment]
Correct me if I'm wrong, but it seems Safari is deprecating support for Extensions.

Edit:
https://developer.apple.com/safari/extensions

Ahh, it seems like they're deprecating an older legacy form of extensions, but there's a new format. Supposedly it needs to be created/built in Xcode? 😞

**Comment by russelldc** [comment]
It seems like the new form of Safari extensions have to be written in Swift, so it'd require a complete re-write, and parallel updates between the JS and Switch code each commit :/ I don't see this happening, @Muffo 

Edit: Actually, since this extension doesn't use a background.js script, it only injects css and js, it might not be such a huge endeavor.

Thoughts, @boukestam?

**Comment by Muffo** [comment]
Yes, that's really painful! I agree it would be a maintenance nightmare unless someone comes up with a framework to _easily_ port extensions from Chrome/Firefox.
